MULTIPLATFORM.md

```diff
@@ -0,0 +1,244 @@
+            classpath "org.jetbrains.kotlin:kotlin-native-gradle-plugin:0.6.2"
+
+        }
+    }
+
+    // Set up compilation dependency repositories for all projects.
+    subprojects {
+        repositories {
+            google()
+            jcenter()
+            mavenCentral()
+        }
+    }
+
+Now all subprojects of the library can use Kotlin plugins.
+
+#### 2.1 Common subproject
+
+The `common` subproject contains a platform-independent code. Let's create `common/src/main/kotlin/common.kt` and
+add some functionality into it:
+
+    // greeting/common/src/main/kotlin/common.kt
+    package org.greeting
+
+    expect class Platform() {
+        val platform: String
+    }
+
+    class Greeting {
+        fun greeting(): String = "Hello, ${Platform().platform}"
+    }
+
+
+Here we create a simple class using `expect`/`actual` paradigm. See details about platform-specific declarations
+[here](https://kotlinlang.org/docs/reference/multiplatform.html#platform-specific-declarations) **TODO: more details here?**
+
+To build this common project add the following snippet in `common/build.gradle`:
+
+    apply plugin: 'kotlin-platform-common'
+
+    // Specify a group and a version of the library to access it in Android Studio.
+    group = 'org.greeting'
+    version = 1.0
+
+    dependencies {
+        // Set up compilation dependency on common Kotlin stdlib
+        compile "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlin_version"
+    }
+
+#### 2.2 Android subproject
+
+The `android` subproject contains platform-dependent implementations of `expect`-declarations we've created in the
+`common` project. Let's implement our `expect`-class:
+
+    // greeting/android/src/main/kotlin/android.kt
+    package org.greeting
+
+    actual class Platform actual constructor() {
+        actual val platform: String = "Android"
+    }
+
+We build this project into a Java-library which can be used by Android Studio project as a dependency. So the content
+of `android/build.gradle` will be the following:
+
+    apply plugin: 'java-library'
+    apply plugin: 'kotlin-platform-jvm'
+
+    // Specify a group and a version of the library to access it in Android Studio.
+    group = 'org.greeting'
+    version = 1.0
+
+    dependencies {
+        // Specify Kotlin/JVM stdlib dependency.
+        implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+
+        // Specify dependency on a common project for Kotlin multiplatform build.
+        expectedBy project(':common')
+    }
+
+#### 2.3 iOS subproject
+
+As well as `android`, this project contains platform-dependent implementations of `expect`-declarations:
+
+    // greeting/ios/src/main/kotlin/ios.kt
+    package org.greeting
+
+    actual class Platform actual constructor() {
+        actual val platform: String = "iOS"
+    }
+
+We build this project into an Objective-C framework using Kotlin/Native compiler. To do this, declare a framework in
```

Comment:
*F*ramework

Reply:
Why should it start with a capital letter? [Here](https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPFrameworks/Frameworks.html#//apple_ref/doc/uid/10000183-SW1) just "framework" is used.

Reply:
Ok. Seems there is no general "rule" for that.
I have expected that `Framework` is the correct wording..